resources/metadata/whitesourceExecuteScan.yaml

```diff
@@ -519,6 +519,21 @@ spec:
                 type: int
               - name: policy_violations
                 type: int
+      - name: reports
+        type: reports
```

Comment:
@OliverNocon how or better to say where do we handle things written to the `./pipeline/stepReports` directory?